src/components/Navigation.tsx

```diff
@@ -269,7 +269,7 @@ const Navigation: FC = () => {
                   <li className="p-side-navigation__item">
                     <a
                       className="p-side-navigation__link"
-                      href="https://discuss.linuxcontainers.org/"
+                      href="https://discourse.ubuntu.com/c/lxd/126"
```

Comment:
I guess this can be:
```suggestion
                      href="https://discourse.ubuntu.com/c/lxd"
```
And then that would redirect automatically?

Reply:
The canonical url is https://discourse.ubuntu.com/c/lxd/126 so I guess we should use it directly instead of relying on the redirict.

Reply:
That is not right, I just changed it.